spec/models/mingle/twitter/tweet_spec.rb

```diff
@@ -9,7 +9,8 @@
 
   describe "#avatar" do
     it 'should be the user image url' do
-      expect(subject.avatar).to eq subject.user_image_url
+      #pending 'need to test that there is an avatar'
+      #expect(subject.avatar).to eq subject.user_image_url
```

Comment:
This should either be removed or uncommented (I think the latter, because it should pass now and it's important that it does for the sake of backwards compatibility).